packages/wonder-blocks-dropdown/src/components/__tests__/action-menu.test.js

```diff
@@ -18,9 +16,18 @@ describe("ActionMenu", () => {
     const onToggle = jest.fn();
     const onChange = jest.fn();
 
+    beforeEach(() => {
+        jest.useFakeTimers();
+    });
+
+    afterEach(() => {
+        jest.runOnlyPendingTimers();
+        jest.useRealTimers();
```

Comment:
I thought that jest switch back automatically, is that not the case?

Reply:
I followed the recommendation here: https://testing-library.com/docs/using-fake-timers/ (It seems that's the case here, specially because we use 3rd party libs like Popper and react-window.

Reply:
TIL.  Thanks for the link to the docs.